File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/items/CheckIfAnyStoredItemsExist.java
package com.lasthopesoftware.bluewater.client.stored.library.items;

import com.lasthopesoftware.bluewater.client.library.repository.LibraryId;
import com.namehillsoftware.handoff.promises.Promise;

public interface CheckIfAnyStoredItemsExist {
	Promise<Boolean> promiseIsAnyStoredItemsOrFiles(LibraryId libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/items/CheckIfAnyStoredItemsExist.kt
package com.lasthopesoftware.bluewater.client.stored.library.items

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId
import com.namehillsoftware.handoff.promises.Promise

interface CheckIfAnyStoredItemsExist {
    fun promiseIsAnyStoredItemsOrFiles(libraryId: LibraryId): Promise<Boolean>
}
